contracts/strategies/kyber/KyberConverterStrategyLogicLib.sol

```diff
@@ -475,10 +482,15 @@ library KyberConverterStrategyLogicLib {
   function needStrategyRebalance(PairBasedStrategyLogicLib.PairState storage pairState, ITetuConverter converter_) external view returns (
     bool needRebalance
   ) {
+    address pool = pairState.pool;
+    uint poolPriceDecimals = IERC20Metadata(pairState.tokenA).decimals();
+    uint poolPriceAdjustment = poolPriceDecimals < 18 ? 10 ** (18 - poolPriceDecimals) : 1;
+    uint poolPrice = KyberLib.getPrice(pool, pairState.tokenB) * poolPriceAdjustment;
```

Comment:
It would be nice to have a comment with explanation what decimals are expected here:
KyberLib.getPrice - returns price with decimals of tokenA (probably it worth to add such comment to KyberLib.getPrice function). 
poolPrice should have same decimals as a price from oracle == 18